lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -138,17 +150,397 @@ sub _prefer_parallel {
     return;
 }
 
+
+sub schedule {
+    my $allocated_worker;
+    my $start_time = time;
+
+    log_debug("+=" . ("-" x 16) . "=+");
+    log_debug("Check if previously dispatched jobs(" . scalar(keys %{$ws_allocated_jobs}) . ") were accepted");
+
+# If in the other tick we allocated some job, let's check meanwhile we got an answer from the worker, saying it have accepted it.
+
+    foreach my $j (keys %{$ws_allocated_jobs}) {
+        my $workerid = is_job_allocated($j);    # ask websocket server over dbus and remove from his 'queue'
+        my $expected_workerid = $ws_allocated_jobs->{$j}->{assigned_worker_id};
+        log_debug("[Job#${j}] Check if was accepted by the assigned worker $expected_workerid :"
```

Comment:
I guess if uninformed users read debug logs, we have a problem. If there is something to do, we should stick to log_warn